opentech/apply/funds/models.py

```diff
@@ -111,9 +109,12 @@ class WorkflowStreamForm(WorkflowHelpers, AbstractStreamForm):  # type: ignore
     class Meta:
         abstract = True
 
-    def get_defined_fields(self):
-        # Only return the first form, will need updating for when working with 2 stage WF
-        return self.forms.all()[0].fields
+    def get_defined_fields(self, stage=None):
```

Comment:
inviting for proposal results in `'Page' object has no attribute 'get_defined_fields`

Reply:
fixed in 7ecfc1d